Class/Mage/MageSpells.cpp

```diff
@@ -109,6 +110,9 @@ MageSpells::MageSpells(Mage* mage) : CharacterSpells(mage), mage(mage) {
     add_spell_group({new ArcanePower(mage)});
     add_spell_group({new Evocation(mage)});
 
+    // Mage armor spell
```

Comment:
Please remove this comment, it is not particularly valuable.

Reply:
OK will do!